multigeiger/webconf.cpp

```diff
@@ -67,7 +106,8 @@ void handleRoot(void) {  // Handle web requests to "/" path.
 }
 
 void configSaved(void) {
-  log(INFO, "Config saved");
+  log(INFO, "Config saved .. restarting");
+  ESP.restart();
```

Comment:
maybe we could detect actual config change somehow? maybe supported by iotWebConf already?

and only restart if changed?

Reply:
Didn't find a way to read the stored configs back from iotwebconf. May be you find one.

Reply:
The `init` method does that via `loadConfig`.

I also found that one can set an own FormValidator.

Reply:
Do you have another version of iotwebconf? There is no loadConfig. There is a configload, but that is private.

Reply:
https://github.com/prampec/IotWebConf/blob/master/src/IotWebConf.cpp#L116

yeah, configLoad is private, but init is public.